exposed-tests/src/test/kotlin/org/jetbrains/exposed/sql/tests/shared/types/DoubleColumnTypeTests.kt

```diff
@@ -0,0 +1,26 @@
+package org.jetbrains.exposed.sql.tests.shared.types
+
+import org.jetbrains.exposed.dao.id.IntIdTable
+import org.jetbrains.exposed.sql.*
+import org.jetbrains.exposed.sql.tests.DatabaseTestsBase
+import org.jetbrains.exposed.sql.tests.shared.assertEquals
+import org.junit.Test
+
+class DoubleColumnTypeTests : DatabaseTestsBase() {
+    object TestTable : IntIdTable("double_table") {
+        val amount = double("amount")
+    }
+
+    @Test
+    fun `test correctly gets data from the DB`() {
+        withTables(TestTable) {
```

Comment:
This tests passes without the fix, understandably because the create DDL is using DOUBLE PRECISION for all DB, which doesn't match your failing use case.

It's ok to leave this test as a control, but please add a second test that uses a REAL column instead, which will fail without the fix. Something like this:
```kt
@Test
fun testInsertAndSelectFromRealColumn() {
    withDb {
        val originalColumnDDL = TestTable.amount.descriptionDdl()
        val realColumnDDL = originalColumnDDL.replace(" DOUBLE PRECISION ", " REAL ")

        // create table with double() column that uses SQL type REAL
        TestTable.ddl
            .map { it.replace(originalColumnDDL, realColumnDDL) }
            .forEach { exec(it) }

        val id = TestTable.insertAndGetId {
            it[amount] = 9.23
       }

        TestTable.selectAll().where { TestTable.id eq id }.singleOrNull()?.let {
            assertEquals(9.23, it[TestTable.amount])
        }

        SchemaUtils.drop(TestTable)
    }
}
```

Also, please change the name of the first test to use camelCase and be a bit more descriptive, anything like for example:
`testInsertAndSelectFromDoubleColumn`. This naming convention more easily allows us to potentially add tests as samples in our KDocs in the future.

Reply:
ah thanks for providing that test case! i wanted to do this, but couldn't find the correctly way to get the column type, thanks!! updating now!